pkg/analyzer/scan.go

```diff
@@ -92,6 +98,23 @@ func ScanK8sDeployObject(kind string, objDataBuf []byte) (common.Resource, error
 	return resourceCtx, nil
 }
 
+func ScanK8sConfigmapObject(kind string, objDataBuf []byte) (common.CfgMapData, error) {
+	var cfgmapCtx common.CfgMapData = make(map[string][]string, 0)
+	obj := ParseConfigMap(bytes.NewReader(objDataBuf))
+
+	//parsed object is a map from ConfigMap's full name (namespace/name) to its data values of interest (list of strings)
+	fullName := obj.ObjectMeta.Namespace + "/" + obj.ObjectMeta.Name
+	data := make([]string, 0)
```

Comment:
```suggestion
	data := []string{}
```
More consistent with the rest of the code

Reply:
done